src/Tokenizer.ts

```diff
@@ -523,33 +550,18 @@ export default class Tokenizer {
             this._state = State.AfterStyle1;
         } else this._state = State.Text;
     }
-    _stateBeforeScript5(c: string) {
```

Comment:
Rather than create a third nearly identical function for title, I consolidated into one reusable function